galsim/phase_screens.py

```diff
@@ -0,0 +1,623 @@
+import numpy as np
+import galsim
+import utilities
+
+class AtmosphericScreen(object):
+    """ An atmospheric phase screen that can drift in the wind and evolves ("boils") over time.  The
+    initial phases and fractional phase updates are drawn from a von Karman power spectrum, which is
+    defined by a Fried parameter that effectively sets the amplitude of the turbulence, and an outer
+    scale that sets the scale beyond which the turbulence power goes (smoothly) to zero.
+
+    @param screen_size   Physical extent of square phase screen in meters.  This should be large
+                         enough to accommodate the desired field-of-view of the telescope as well as
+                         the meta-pupil defined by the wind speed and exposure time.  Note that
+                         the screen will have periodic boundary conditions, so the code will run
+                         with a smaller sized screen, though this may introduce artifacts into PSFs
+                         or PSF correlations functions. Note that screen_size may be tweaked by the
+                         initializer to ensure screen_size is a multiple of screen_scale.
+    @param screen_scale  Physical pixel scale of phase screen in meters.  A fraction of the Fried
+                         parameter is usually sufficiently small, but users should test the effects
+                         of this parameter to ensure robust results. [Default: half of r0_500]
+    @param altitude      Altitude of phase screen in km.  This is with respect to the telescope, not
+                         sea-level.  [Default: 0.0]
+    @param time_step     Interval to use when advancing the screen in time in seconds.
+                         [Default: 0.03]
+    @param r0_500        Fried parameter setting the amplitude of turbulence; contributes to "size"
+                         of the resulting atmospheric PSF.  Specified at wavelength 500 nm, in units
+                         of meters.  [Default: 0.2]
+    @param L0            Outer scale in meters.  The turbulence power spectrum will smoothly
+                         approach a constant at scales larger than L0.  Set to `None` or `np.inf`
+                         for a power spectrum without an outer scale.  [Default: 25.0]
```

Comment:
L0=25m is admittedly a bit arbitrary, but in the expected range which I've seen anywhere from 10 to 100 meters.